_layouts/ontology_detail.html

```diff
@@ -70,9 +70,44 @@ <h1>
         <a href="http://www.ebi.ac.uk/ols/ontologies/{{page.id}}" class="btn btn-outline-primary" target="_blank" rel="noopener">
           OLS
         </a>
-        <a href="https://bioportal.bioontology.org/ontologies/{{ page.id | make_bioportal_id }}" class="btn btn-outline-primary" target="_blank" rel="noopener">
-          BioPortal
-        </a>
+        {% assign skip_bioportal = false %}
+        {% for browser in page.browsers %}
+          {% if browser.label.upcase == "BIOPORTAL" %}
+            {% assign skip_bioportal = true %}
+          {% endif %}
+        {% endfor %}
+
+        {% unless skip_bioportal %}
+          {% assign bioportal_id = page.id | upcase %}
+          {% if bioportal_id == 'FBBT' %}
+            {% assign bioportal_id = 'FB-BT' %}
+          {% elsif bioportal_id == 'RO' %}
+            {% assign bioportal_id = 'OBOREL' %}
+          {% elsif bioportal_id == 'APOLLO_SV' %}
+            {% assign bioportal_id = 'APOLLO-SV' %}
+          {% elsif bioportal_id == 'TRANS' %}
+            {% assign bioportal_id = 'PTRANS' %}
+          {% elsif bioportal_id == 'WBLS' %}
+            {% assign bioportal_id = 'WB-LS' %}
+          {% elsif bioportal_id == 'FBDV' %}
+            {% assign bioportal_id = 'FB-DV' %}
+          {% elsif bioportal_id == 'WBBT' %}
+            {% assign bioportal_id = 'WB-BT' %}
+          {% elsif bioportal_id == 'WBPHENOTYPE' %}
+            {% assign bioportal_id = 'WB-PHENOTYPE' %}
+          {% elsif bioportal_id == 'TO' %}
+            {% assign bioportal_id = 'PTO' %}
+          {% elsif bioportal_id == 'FBCV' %}
+            {% assign bioportal_id = 'FB-CV' %}
+          {% elsif bioportal_id == 'MOD' %}
+            {% assign bioportal_id = 'PSIMOD' %}
+          {% elsif bioportal_id == 'PSO' %}
+            {% assign bioportal_id = 'PLANTSO' %}
+          {% endif %}
+          <a href="https://bioportal.bioontology.org/ontologies/{{ bioportal_id }}" class="btn btn-outline-primary" target="_blank" rel="noopener">
+            BioPortal
+          </a>
+        {% endunless %}
```

Comment:
Wait the way you do it, if a bioportal link is manually set, wont they now see _no button_ at all?

Reply:
Don't believe so -- 

Lines 73 through 78 set a local `skip_bioportal` variable. If it's false, then lines 80-110 render a BioPortal button, but if it's true, then that's skipped. Regardless, 111-115  render custom buttons for each set of values defined in the ontology's front matter.

https://github.com/OBOFoundry/OBOFoundry.github.io/blob/c6e650d3012c8cbc0baca2a49ba7e9299f88ff4c/_layouts/ontology_detail.html#L111-L115

Reply:
eg here's my local render of PRO

![Screenshot 2024-11-25 at 1 22 17 PM](https://github.com/user-attachments/assets/5c67889a-9d07-4678-9863-b981ae5d7460)